tests/ts_util.erl

```diff
@@ -315,6 +315,7 @@ get_valid_big_data(N) ->
         N + 0.1,
         get_bool(X),
         N + 100000,
+        get_varchar(),
```

Comment:
Will this break inserts and result set assertions for other tests using this shared table def?